crates/kubelet/src/store/oci/file.rs

```diff
@@ -336,12 +330,13 @@ mod test {
         let fake_ref = Reference::try_from("foo/bar:2.0")?;
         let scratch_dir = create_temp_dir();
         let store = FileStore::new(fake_client.clone(), &scratch_dir.path);
-        let module_bytes_orig = store.get(&fake_ref, None).await?;
+        let policy = PullPolicy::parse_with_ref(None, Some(fake_ref.clone()))?;
```

Comment:
I wonder if this might be better moved to the `Container` tests?  That way you can have a set of tests that focus on the correct resolution of the pull policy, and when you get to the store tests, you can take it as read that we resolve pull policies correctly and can focus on testing that we implement each policy correctly - no need for the "None" tests.

Reply:
Yes, I wasn't sure about removing those Store tests entirely as I wasn't sure if there was other functionality being tested there as well.